### ISSUE TITLE
[Improvement] glpi-agent-deployment.vbs and local MSI

### ISSUE THREAD

**Issue by MarcSamD** [glpi-project/glpi-agent#429]
### Describe the bug

With fusioninventory-agent-deployment.vbs, it was possible to use `SetupLocation = "."` for executing the .exe in the same folder as the script in relative path.
With glpi-agent-deployment.vbs and the use of MSI, this doesn't work anymore because MsiExec doesn't like to have `".\"` as the current path (I guess that it is then looking into the system folder where MsiExec.exe itself is located) :
![image](https://github.com/glpi-project/glpi-agent/assets/70358310/e6f5b68a-a5cd-4a42-b833-14e987e3622c)

But it is working simply without path:
![image](https://github.com/glpi-project/glpi-agent/assets/70358310/77675d88-217b-4e87-8598-c3bbdf3df339)

Would you mind to edit the following part of the vbs to support either `SetupLocation = ""` or `SetupLocation = "."` ?
```
   Else
      ShowMessage("Running: MsiExec.exe " & strInstallOrRepair & " """ & SetupLocation & "\" & Setup & """ " & SetupOptions)
      WshShell.Run "MsiExec.exe " & strInstallOrRepair & " """ & SetupLocation & "\" & Setup & """ " & SetupOptions, 0, True
      ShowMessage("Deployment done!")
   End If
```

### Operating system

Windows

**Comment by g-bougard** [comment]
Hi @MarcSamD 

thank you for your report. I'll make some tests myself to find the best solution between yours and any other I can think about.

**Comment by kintaro1981** [comment]
I would like to install from same directory too.
(I have some win7 and the wget inside the vbs doesn't work with win7)

@MarcSamD do you found a solution?

**Comment by g-bougard** [comment]
> I would like to install from same directory too. (I have some win7 and the wget inside the vbs doesn't work with win7)
> 
> @MarcSamD do you found a solution?

You should open another issue for the downloading problem not working on win7. If you know a more reliable method to download from this windows version, don't hesitate to share it.

I still didn't take time to investigate this issue problem, but I'll do before the next release coming in few weeks.

**Comment by kintaro1981** [comment]
I don't want to waste your (and my) time with Win7. I think there aren't many requests about it, and with the local solution below the problem is solved for win7 too.

I solved editing VBS file in this way:

I replaced this:
```
   Else
      ShowMessage("Running: MsiExec.exe " & strInstallOrRepair & " """ & SetupLocation & "\" & Setup & """ " & SetupOptions)
      WshShell.Run "MsiExec.exe " & strInstallOrRepair & " """ & SetupLocation & "\" & Setup & """ " & SetupOptions, 0, True
      ShowMessage("Deployment done!")
   End If
```

```
   Else
      ShowMessage("Running: MsiExec.exe " & strInstallOrRepair & " """ & SetupLocation & "" & Setup & """ " & SetupOptions)
      WshShell.Run "MsiExec.exe " & strInstallOrRepair & " """ & SetupLocation & "" & Setup & """ " & SetupOptions, 0, True
      ShowMessage("Deployment done!")
   End If
```
I removed the backslash \ inside the double quotes beetween SetupLocation and Setup, probably I could remove the whole `& "\" `

In this way I got:

```
C:\Users\Administrator\Desktop\installa glpi-agent>cscript glpi-agent-installa.vbs
Microsoft (R) Windows Script Host Versione 5.8
Copyright (C) Microsoft Corporation 1996-2001. Tutti i diritti riservati.

System architecture detected: x64
Setup architecture detected: x64
Installation needed: 1.5
Running: MsiExec.exe /i "GLPI-Agent-1.5-x64.msi" /quiet RUNNOW=1 SERVER=http://glpiserver/plugins/glpiinventory DEBUG=1 AGENTMONITOR=1 ADD_FIREWALL_EXCEPTION=1
Deployment done!

```

**Comment by g-bougard** [comment]
Hi @MarcSamD & @kintaro1981 

I included a fix in a PR #507. You can try it by downloading it directly here: https://raw.githubusercontent.com/glpi-project/glpi-agent/9fbc65fce120a9fca84d2912e87e4fa010ec25e0/contrib/windows/glpi-agent-deployment.vbs

After you tried it, can you comment the PR and/or this issue ?

Thank you
